### ISSUE TITLE
iOS and Android not part of examples for WCAG, should be deleted 2.5.2: Pointer Cancellation

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#706]
There are possible techniques suggested for 2.5.2: Pointer Cancellation:

https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html#techniques

> @@ Activating a control using the up-Event in HTML, iOS and Android.

As the scope of WCAG is not native and we're not providing examples for native / iOS / Android we should not suggest we will add techniques for them.

**Comment by mbgower** [comment]
I'll make a PR for this and the other issue you mentioned.

**Comment by mbgower** [comment]
https://github.com/w3c/wcag/pull/711

**Comment by jake-abma** [comment]
see it's fixed
